### ISSUE TITLE
Infinity-Bug

### ISSUE THREAD

**Issue by FlashyZip** [CompactMods/CompactMachines#286]
Hello, 
I found a bug while creating Compact Machine Walls, I placed the Iron-Block with the redstone not in the middle and then, the compact machine wall comes infinitly and dont stop. It doesnt help if I removed the field projectors.

**Comment by davenonymous** [comment]
Please try to be more precise. I have no clue what your issue is.
How to report bugs: https://vazkii.us/br101/

**Comment by FlashyZip** [comment]
I'm using Compact Machines 3 - 1.12.2-3.0.3-b169 in the Modpack Direwolf2.0 1.3.0.
I'm playing this Modpack on a Server. (Bukkit-Server if this matter)

I placed the Miniaturization Field Projectors with a crafting field of 3x3x3.
Then I want to craft Compact Machine Walls and placed the Iron-Block with the redstone on it. Instead placing the Iron-Block with the redstone in the middle, I placed it on the South relative to the middle.

Legend: X - Miniaturization Field, C - Iron Block with Redstone on it

XXX
XXX
XCX

Then instead of throw just 16 Compact Machine Walls it throws infinitly Blocks, since 20min it dont stop.

**Comment by davenonymous** [comment]
That sounds like an interesting bug. I'll try to reproduce this.
Bukkit-Server might very well matter. Do you have some sort of permissions or chunk protection system in place?

**Comment by FlashyZip** [comment]
Yeah, I was on a claimed chunk. I dont know, whats the name of the mod, but I can claim chunks with a Gold Shovel.

**Comment by FlashyZip** [comment]
But yet I see, different things, then before. 
When I try to craft this Compact Machine Walls, I but them in the crafting field like in JEI, but nothing happens, except a smoke, that showing all over the time, when a right craft item was in place. The Field projectors show the valid recipe of them, too.
But when I throw a redstone to start the crafting process into the crafting field, the redstone gets away, but nothing happens.

And at the next day I fixed this by just setting one field projector at the same place. Then it stopped blobbing items from nowhere. I tested it the day before, but this doesnt help. 
Maybe a bug with the date?

And now I cannot reproduce this, at the same location as before.

And thats the first time using compact machines with this client.

**Comment by Vercci** [comment]
Just had this happen also. Hope this helps,

Playing on a server running SpongeForge, and with GriefPrevention as the world claim system. Server was running Enigmatica 2 Expert, version 1.34 at the time of the machine being crafted.

Tried crafting compact machine walls with a 3x3 projector setup. Throwing redstone at the iron block + redstone caused the extra thrown redstone to fizzle, have black dust particles appear but nothing happen to the crafting recipe. We thought the claim system was interfering because half of the projectors were in a claim so we moved all of them out of the claim, but still the same result.

Thought it may be an issue with too big a projection field. Found a reddit thread where I believe you said you were gonna disable the 1x1 field because people kept having issues crafting the walls there. I then tried making a 5x5, still outside any claims. This time the recipe appeared to work, however, when the beam and box got really small and started ejecting the item, it never stopped.

https://i.imgur.com/Wba3kGY.jpg

Added the lava pit to clear excess items. It is still ejecting items, will try to let you know if it continues after a scheduled restart. I didn't get Flashy Zip's solution of re-placing the projectors to work, they just continue to show the small beams converging at hte point the items are spawning. In fact, while right clicking the projectors, I get the message "Currently crafting: Compact Machine Wall (progress: 101.0%)" if that helps.

[edit] Forgot to add the edit. Re-placing the projectors didn't stop the crafting but we believe putting a block on the "block" above where the crafting was (and where we weren't able to place blocks while it was spitting out items) stopped it. Either that, or setting off tnt with a flint and steel we weren't too sure at the time.
